libs/ledger-live-common/src/network.ts

```diff
@@ -34,18 +81,28 @@ export const errorInterceptor = (error: AxiosError<any>): AxiosError<any> => {
     }
     log(
       "network-error",
-      `${status} ${method} ${baseURL || ""}${url}: ${errorToThrow.message}`,
+      `${status} ${method} ${baseURL || ""}${url} (${(
+        Date.now() - startTime
+      ).toFixed(0)}ms): ${errorToThrow.message}`,
       getEnv("DEBUG_HTTP_RESPONSE") ? { data: data } : {}
     );
     throw errorToThrow;
   } else if (error.request) {
-    log("network-down", `DOWN ${method} ${baseURL || ""}${url}`);
+    log(
+      "network-down",
+      `DOWN ${method} ${baseURL || ""}${url} (${(
+        Date.now() - startTime
+      ).toFixed(0)}ms)`
+    );
     throw new NetworkDown();
   }
   throw error;
 };
 
-axios.interceptors.response.use(undefined, errorInterceptor);
+axios.interceptors.request.use(requestInterceptor);
+
+// $FlowFixMe LLD raise issues here
```

Comment:
Is that comment needed anymore?
(Maybe not after https://github.com/LedgerHQ/ledger-live/pull/3359 ?)